### DISCUSSION TITLE
Dear author, I hope to are some 3d charts, such as 3d pie charts, etc

### DISCUSSION THREAD

**Discussion by zeMinng** [graphieros/vue-data-ui#65] [Ideas]
Dear author, I hope to are some 3d charts, such as 3d pie charts, etc. thanks

**Comment by graphieros** [comment]
Dear @zeMingGit, thank you for your message.

It is not recommended to use 3d for data visualization in general, because 3d distorts proportions and is harder to understand for users.

The only 3d chart available for now is [VueUi3dBar](https://vue-data-ui.graphieros.com/docs#vue-ui-3d-bar), as it stays relatively readable and simple. 3d pie charts perspective distorts proportions and makes it harder to understand angles.

I am committed to providing components that are a compromise between effectiveness and esthetics. It is my current understanding that most 3d charts are more esthetic than effective.

Feel free to prove me wrong, with links to cool 3d charts that are more effective  than 2d to convey information:)

You can check out [this article](https://medium.com/@purohitpraveen/data-vizualization-why-3d-charts-are-a-terrible-idea-32657fbb928e) that says it better than I do.

Cheers

**Reply by zeMinng** [reply]
tanks.
Cheers
